src/metadata.js

```diff
@@ -119,9 +119,17 @@ Metadata.extractResponse = curry((encKey, res) => {
   if (res === null) {
     return res;
   } else {
+    let parseOrLog = (str) => {
+      try {
+        return JSON.parse(str);
+      } catch (e) {
+        console.log(str);
```

Comment:
Good idea to log this out. Maybe prefix it with a message for context? Like "Unable to parse metadata contents: ..."

Reply:
Good point, thanks